packages/launcher/src/renderer/launcher/settings/SettingsScreen.js

```diff
@@ -34,9 +40,26 @@ export default class SettingsScreen extends Component<{}, State> {
   }
 
   onSelectMenuItem = (key: MenuKey) => {
-    this.setState({
-      openView: key,
-    })
+    switch (key) {
+      case 'docs':
+        shell.openExternal('https://mainframe.com/developers/')
+        break
+      case 'help':
+        shell.openExternal('https://community.mainframe.com/')
+        break
+      case 'feedback':
+        shell.openExternal('https://mainframe.com/contact/')
+        break
+      case 'update':
+        shell.openExternal(
+          'https://github.com/MainframeHQ/js-mainframe/releases',
```

Comment:
Could you point it to `MainframeHQ/mainframe-os` instead as we'll rename the repository before we publish v0.2 please?